charts/datafold/charts/worker/templates/deployment.yaml

```diff
@@ -40,6 +41,8 @@ spec:
             - configMapRef:
                 name: {{ include "datafold.configmap" . }}
           env:
+            - name: QUEUES
+              value: "celery,api,ci,interactive,freshpaint,lineage,catalog,singletons,alerts"
```

Comment:
We already have defined this here: https://github.com/datafold/helm-charts/blob/14b0590905a89edef82a09672d4671fefaa9f475/charts/datafold/charts/worker/templates/_helpers.tpl#L53C1-L56

Reply:
Also using the annotation is deprecated: https://github.com/datafold/helm-charts/blob/14b0590905a89edef82a09672d4671fefaa9f475/charts/datafold/charts/nginx/templates/ingress.yaml#L9